src/test/java/core/basesyntax/model/UserTest.java

```diff
@@ -0,0 +1,104 @@
+package core.basesyntax.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class UserTest {
```

Comment:
Please remove this class. It tests itself and has no connection to code of program

Reply:
i removed this class , place it i created RegistrationServiceTest